### PR TITLE
todo app react1-week3

### PR DESCRIPTION
**Pull request by hamoudi-hamoudi**

### CODE REVIEW

File: react/react1/week3/src/pages/NotePage.jsx [hunk 1]
```diff
@@ -0,0 +1,100 @@
...
+    const { name, value } = e.target;
+    setNoteValue((prev) => ({ ...prev, [name]: value }));
+  };
+
+  const addTodo = (e) => {
+    e.preventDefault();
+    const { description, deadline } = noteValue;
+    const now = new Date();
+    const comareDeadLinre = now < new Date(deadline);
+    if (description.trim() === "" || !comareDeadLinre) {
```
Comment (YurijPaharev) [inline react/react1/week3/src/pages/NotePage.jsx]: what if description is null?

File: react/react1/week3/src/pages/NotePage.jsx [hunk 2]
```diff
@@ -0,0 +1,100 @@
...
+  const handleNoteValue = (e) => {
+    const { name, value } = e.target;
+    setNoteValue((prev) => ({ ...prev, [name]: value }));
+  };
+
+  const addTodo = (e) => {
+    e.preventDefault();
+    const { description, deadline } = noteValue;
+    const now = new Date();
+    const comareDeadLinre = now < new Date(deadline);
```
Comment (YurijPaharev) [inline react/react1/week3/src/pages/NotePage.jsx]: I think as a precaution you should compare them by number

**Review by YurijPaharev — APPROVED** [review]
Super!!!

**Comment by hamoudi-hamoudi** [comment]
Thanks Yuri for the review. going to fix the code 🥇

**Pull request merged**